public/locales/en/translation.json

```diff
@@ -347,7 +347,7 @@
     "alert": "You have successfully submitted your retroactive certification.",
     "alert-returning": "You have successfully submitted your retroactive certification. No further action is required.",
     "header1": "Retroactive Certification Confirmation",
-    "p1": "Your retroactive certification has been submitted. Your confirmation number is <strong>{{confirmationNumber}}</strong>.",
+    "p1": "<p>Your retroactive certification has been submitted. Your confirmation number is <strong>{{confirmationNumber}}</strong>.</p><p>Keep this number for your records.</p>",
```

Comment:
how did this work? officially you can't have second-level tags in i18next-react Trans components, so this <strong>tag</strong> tag shouldn't work, but i see it in your screenshot...

Reply:
Oh interesting! I have no idea 😂 Maybe the bold text won't display in other browsers?

Reply:
I see this in the documentation: `<bold>bold<i>italic</i></b> // no inner elements - only strings!`, but maybe the problem is that `<i>` isn't supported?

Reply:
Why not just add a new entry to the file for the second paragraph?

Reply:
Oh, I am! It'll be ready in a minute!